pkg/kine/server/maintenance.go

```diff
@@ -19,8 +19,9 @@ func (s *KVServerBridge) Status(ctx context.Context, r *etcdserverpb.StatusReque
 		return nil, err
 	}
 	return &etcdserverpb.StatusResponse{
-		Header: &etcdserverpb.ResponseHeader{},
-		DbSize: size,
+		Header:  &etcdserverpb.ResponseHeader{},
+		DbSize:  size,
+		Version: s.emulatedEtcdVersion,
```

Comment:
The emulatedEtcdVersion  is returned on status calls, giving an indication to the api-server whether the watch progress requests are supported or not.